src/modules/formBuilder/components/FormEditorItemProperties.tsx

```diff
@@ -0,0 +1,182 @@
+  item,
+  onChangeProperty,
+}) => {
+  return (
+    <>
+      <Typography>Properties</Typography>
+      <Stack
+        gap={2}
+        component='form'
+        onSubmit={() => {
+          console.log('submitted'); // TODO(#6103)
+        }}
+        sx={{ maxWidth: MAX_INPUT_AND_LABEL_WIDTH }}
+      >
+        <TextInput
+          label='Link ID'
+          value={item.linkId}
+          onChange={(e) => {
+            onChangeProperty('linkId', e.target.value);
+          }}
+          helperText='Unique ID for this form item'
+        />
+        <LabeledCheckbox
+          label='Assessment date'
+          checked={!!item.assessmentDate}
+          onChange={(e) =>
+            onChangeProperty(
+              'assessmentDate',
+              (e.target as HTMLInputElement).checked
+            )
+          }
+        />
+        <LabeledCheckbox
+          label='Required'
+          checked={item.required}
+          onChange={(e) =>
+            onChangeProperty('required', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <TextInput
+          label='Label'
+          value={item.text}
+          onChange={(e) => {
+            onChangeProperty('text', e.target.value);
+          }}
+        />
+        <TextInput
+          label='Helper text'
+          value={item.helperText || ''}
+          onChange={(e) => {
+            onChangeProperty('helperText', e.target.value);
+          }}
+        />
+        <TextInput
+          label='Brief label'
+          value={item.briefText || ''}
+          onChange={(e) => {
+            onChangeProperty('briefText', e.target.value);
+          }}
+          helperText="Label to display when the item is referenced briefly, such as in an Autofill dialog box. If not specified, the item's normal label text is shown."
+        />
+        <TextInput
+          label='Read-only label'
+          value={item.readonlyText || ''}
+          onChange={(e) => {
+            onChangeProperty('readonlyText', e.target.value);
+          }}
+          helperText="Label to display when the item is shown in a read-only form. If not specified, the item's normal label text is shown."
+        />
+        <LabeledCheckbox
+          label='Read-only'
+          checked={item.readOnly}
+          onChange={(e) =>
+            onChangeProperty('readOnly', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <LabeledCheckbox
+          label='Hidden'
+          checked={item.hidden}
+          onChange={(e) =>
+            onChangeProperty('hidden', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <TextInput
+          // TODO(#5776)
+          label='Mapping for custom field key'
+          value={item.mapping?.customFieldKey || ''}
+          onChange={(e) => {
+            onChangeProperty('mapping', { customFieldKey: e.target.value });
+          }}
+        />
+        <LabeledCheckbox
+          label='Warn if empty'
```

Comment:
* Helper text: "If checked, user will see a warning if they don't provide an answer to this question."
* Maybe this and `required` should be grouped together somehow

Reply:
I'll put them next to each other for now, but let's discuss more w/ design tomorrow about ordering and grouping these properties.